src/app/example_local.ml

```diff
@@ -102,6 +102,9 @@ let example () =
       download
         "https://storage.googleapis.com/hammerlab-biokepi-data/precomputed/b37decoy_20160927.tgz"
         ~in_directory:(biokepi_work#mount // "workdir/reference-genome");
+      download
+        "https://storage.googleapis.com/hammerlab-biokepi-data/precomputed/b37_20161007.tgz"
+        ~in_directory:(leading_extra_nfs // "workdir/reference-genome");
```

Comment:
This doesn't build for me; it should be `biokepi_work#mount`.

I'll while merging.